float64 throttle # [0, 1] percentage of throttle to apply
float64 brake # [0, 1] percentage of brake to apply
float64 angular_xy # steering rate, rad/s
float64 linear_z # change in altitude, m/s
